File: frameworks/ruby-rails/app/models/comment.rb
class Comment < ApplicationRecord
  self.table_name = 'benchmark.tb_comment'
  self.primary_key = 'pk_comment'

  belongs_to :author, class_name: 'User', foreign_key: 'fk_author'
  belongs_to :post, class_name: 'Post', foreign_key: 'fk_post'
end

File: frameworks/php-laravel/app/Models/Comment.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class Comment extends Model
{
    use HasFactory;

    protected $primaryKey = "pk_comment";
    protected $keyType = "int";
    public $incrementing = true;

    protected $fillable = [
        "id",
        "content",
        "fk_post",
        "fk_author",
        "created_at"
    ];

    protected $table = "benchmark.tb_comment";
    public $timestamps = false;

    public function post(): BelongsTo
    {
        return $this->belongsTo(Post::class, "fk_post", "pk_post");
    }

    public function author(): BelongsTo
    {
        return $this->belongsTo(User::class, "fk_author", "pk_user");
    }
}